### DISCUSSION TITLE
Ask for some place names in name quest?

### DISCUSSION THREAD

**Discussion by matkoniecz** [streetcomplete/StreetComplete#2727] [Ideas]
I just enabled asking about `place` names in my fork and was thinking does it make sense to do it in general StreetComplete.

For example 7% of mapped `place=village` is without names - see https://taginfo.openstreetmap.org/tags/place=village#combinations (though I suspect that vast majority of them is simply mapped incorrectly - but detecting that also would be helpful)

Over 11% of `place=hamlet` is somehow without `name` - https://taginfo.openstreetmap.org/tags/place=hamlet#combinations

Also, 1% of `place=city` is without `name` - https://taginfo.openstreetmap.org/tags/place=city#combinations

Asking about things like `place=island` may be out of scope like asking about names of rivers a hard to survey, but `place=hamlet` and higher (or maybe `place=village` and larger) seems like a viable target to me.

**Comment by mnalis** [comment]
Makes sense to me, `place=village` and higher for sure. I'm not sure about `place=hamlet`, few places that come to my mind down here might only be known to inhabitants (assuming it has a name at all), and  passer-bys probably won't be able to answer unless they ask someone (that might be country-specific, though, so I'm not against it if others see the use of it - it is quite rare, so won't be spammy or big deal in any case).

I'd also include `place=island`: either island is big and well known and this is simple omission that even causal tourist will be able to answer correctly; or it is small uninhabited island that only people with personal boats will reach (and they tend to be bunch of people who know archipelagos they move in, so will be well aware on which island they are going to / near). 

Rivers are somewhat problematic. While big ones are probably all named anyway, small ones might be problematic to answer (unless one is local inhabitant, of course), as likely only place with river name might be the bridge over it (at least around here). So if it is going to be asked, it should probably be asked only in vicinity of the bridge going over it.

**Comment by westnordost** [comment]
I suspect that many villages and hamlets that currently do not have a name do actually not have a name at all. How many of these unnamed villages are in the savanna of Africa? You know about the HOT mappers and them mapping all those accumulations of huts - I can imagine that people also added `place=village` etc to these.

**Reply by mnalis** [reply]
Could be - I cannot verify it with [overpass-turbo](https://overpass-turbo.eu/s/16cA) as it seems to be too heavy a query (even when I run it on just small part of Croatia, which should not have many - if any - results). If the place does not have a name in reality, shouldn't it be tagged with `noname=yes` anyway (which I assume the quest would mark if user specified that the place really has no name)?
